aou/aou-rstudio-bioconductor/rstudio/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Run wondershaper to limit network bandwidth to 8mbps down and 4mbps up as
+# a different user than RStudio user which is set in the /init script.
+su-exec root wondershaper eth0 8096 4048 && nginx -g 'daemon on;'
+
+# Init will handle it's own UID and GID mapping
+/init
```

Comment:
```suggestion
su-exec $UID:$GID /init # please replace with appropriate UID/GID
```

Reply:
This still needs to run as `root` since it runs `S6` in `/init` and sets `UID` and `GID` in there as a non-root user called `rstudio`

Reply:
Added some comments to explain